src/BlazorWebView/src/Maui/BlazorWebView.cs

```diff
@@ -30,6 +30,20 @@ public BlazorWebView()
 		/// </summary>
 		public string? HostPage { get; set; }
 
+		/// <summary>
+		/// Bindable property for StartPath
```

Comment:
```suggestion
		/// Bindable property for <see cref="StartPath"/>.
```

(Check that this is the right doc syntax 😄 )